include/region_map.h

```diff
@@ -1,227 +1,314 @@
-    REGION_MAP_WEEPTH_CHAMBER,
-    REGION_MAP_DILFORD_CHAMBER,
-    REGION_MAP_SCUFIB_CHAMBER,
-    REGION_MAP_RIXY_CHAMBER,
-    REGION_MAP_VIAPOIS_CHAMBER,
-    REGION_MAP_EMBER_SPA,
-    REGION_MAP_SPECIAL_AREA,
-    REGION_MAP_AQUA_HIDEOUT,
-    REGION_MAP_MAGMA_HIDEOUT,
-    REGION_MAP_MIRAGE_TOWER,
-    REGION_MAP_BIRTH_ISLAND_2,
-    REGION_MAP_FARAWAY_ISLAND,
-    REGION_MAP_ARTISAN_CAVE,
-    REGION_MAP_MARINE_CAVE,
-    REGION_MAP_UNDERWATER_MARINE_CAVE,
-    REGION_MAP_TERRA_CAVE,
-    REGION_MAP_UNDERWATER_TERRA_CAVE,
-    REGION_MAP_UNDERWATER_UNK1,
-    REGION_MAP_UNDERWATER_UNK2,
-    REGION_MAP_DESERT_UNDERPASS,
-    REGION_MAP_ALTERING_CAVE_2,
-    REGION_MAP_NAVEL_ROCK2,
-    REGION_MAP_TRAINER_HILL
+    MAPSEC_LITTLEROOT_TOWN,
+    MAPSEC_OLDALE_TOWN,
+    MAPSEC_DEWFORD_TOWN,
+    MAPSEC_LAVARIDGE_TOWN,
+    MAPSEC_FALLARBOR_TOWN,
+    MAPSEC_VERDANTURF_TOWN,
+    MAPSEC_PACIFIDLOG_TOWN,
+    MAPSEC_PETALBURG_CITY,
+    MAPSEC_SLATEPORT_CITY,
+    MAPSEC_MAUVILLE_CITY,
+    MAPSEC_RUSTBORO_CITY,
+    MAPSEC_FORTREE_CITY,
+    MAPSEC_LILYCOVE_CITY,
+    MAPSEC_MOSSDEEP_CITY,
+    MAPSEC_SOOTOPOLIS_CITY,
+    MAPSEC_EVER_GRANDE_CITY,
+    MAPSEC_ROUTE_101,
+    MAPSEC_ROUTE_102,
+    MAPSEC_ROUTE_103,
+    MAPSEC_ROUTE_104,
+    MAPSEC_ROUTE_105,
+    MAPSEC_ROUTE_106,
+    MAPSEC_ROUTE_107,
+    MAPSEC_ROUTE_108,
+    MAPSEC_ROUTE_109,
+    MAPSEC_ROUTE_110,
+    MAPSEC_ROUTE_111,
+    MAPSEC_ROUTE_112,
+    MAPSEC_ROUTE_113,
+    MAPSEC_ROUTE_114,
+    MAPSEC_ROUTE_115,
+    MAPSEC_ROUTE_116,
+    MAPSEC_ROUTE_117,
+    MAPSEC_ROUTE_118,
+    MAPSEC_ROUTE_119,
+    MAPSEC_ROUTE_120,
+    MAPSEC_ROUTE_121,
+    MAPSEC_ROUTE_122,
+    MAPSEC_ROUTE_123,
+    MAPSEC_ROUTE_124,
+    MAPSEC_ROUTE_125,
+    MAPSEC_ROUTE_126,
+    MAPSEC_ROUTE_127,
+    MAPSEC_ROUTE_128,
+    MAPSEC_ROUTE_129,
+    MAPSEC_ROUTE_130,
+    MAPSEC_ROUTE_131,
+    MAPSEC_ROUTE_132,
+    MAPSEC_ROUTE_133,
+    MAPSEC_ROUTE_134,
+    MAPSEC_UNDERWATER_124,
+    MAPSEC_UNDERWATER_125,
+    MAPSEC_UNDERWATER_126,
+    MAPSEC_UNDERWATER_127,
+    MAPSEC_UNDERWATER_SOOTOPOLIS,
+    MAPSEC_GRANITE_CAVE,
+    MAPSEC_MT_CHIMNEY,
+    MAPSEC_SAFARI_ZONE,
+    MAPSEC_BATTLE_FRONTIER,
+    MAPSEC_PETALBURG_WOODS,
+    MAPSEC_RUSTURF_TUNNEL,
+    MAPSEC_ABANDONED_SHIP,
+    MAPSEC_NEW_MAUVILLE,
+    MAPSEC_METEOR_FALLS,
+    MAPSEC_METEOR_FALLS2,
+    MAPSEC_MT_PYRE,
+    MAPSEC_AQUA_HIDEOUT_OLD,
+    MAPSEC_SHOAL_CAVE,
+    MAPSEC_SEAFLOOR_CAVERN,
+    MAPSEC_UNDERWATER_128,
+    MAPSEC_VICTORY_ROAD,
+    MAPSEC_MIRAGE_ISLAND,
+    MAPSEC_CAVE_OF_ORIGIN,
+    MAPSEC_SOUTHERN_ISLAND,
+    MAPSEC_FIERY_PATH,
+    MAPSEC_FIERY_PATH2,
+    MAPSEC_JAGGED_PASS,
+    MAPSEC_JAGGED_PASS2,
+    MAPSEC_SEALED_CHAMBER,
+    MAPSEC_UNDERWATER_SEALED_CHAMBER,
+    MAPSEC_SCORCHED_SLAB,
+    MAPSEC_ISLAND_CAVE,
+    MAPSEC_DESERT_RUINS,
+    MAPSEC_ANCIENT_TOMB,
+    MAPSEC_INSIDE_OF_TRUCK,
+    MAPSEC_SKY_PILLAR,
+    MAPSEC_SECRET_BASE,
+    MAPSEC_SS_TIDAL,
```

Comment:
???????????

Reply:
I'm saying that calling it MAPSEC_SS_TIDAL is a misnomer because there are many maps which fall under this index, and not all of them are in SS Tidal.

Reply:
This mapsec is used only when you are on the SS TIDAL, to create the illusion that the boat is moving.

Reply:
If you check something like Advance-Map, maps like the Union Room and Cable Club fall under this mapsec, as well as contest awards and a bunch of test maps.